scss/_icon-family-calc.scss

```diff
@@ -0,0 +1,44 @@
+@use "sass:math";
+@use "sass:map";
+@use "sass:list";
+
+$all-icon-sizes: (
+	16: "icons-16-outline",
+	20: "icons-20-outline",
+	24: "icons-24-outline",
+	32: "icons-32-outline",
+	48: "icons-48-outline",
+	64: "icons-64-outline"
+);
```

Comment:
we additionally the 20, 24, 32, 48 and 64 sizes as filled styles.

Reply:
We postpone this, and I will ask UX about it